tests/test_result_to_dict.py

```diff
@@ -200,6 +203,17 @@ def test_callback(self):
         data = result_to_dict(json_value_without_str_obj, callback=convert_return_none)
         self.assertDictEqual(data, json_value_without_str_obj)
 
+    @mongomock.patch(servers=(('serverdb.example.com', 27017), ))
+    def test_pymongo_to_dict(self):
+        client = pymongo.MongoClient('serverdb.example.com')

```

Comment:
will expect to see similar tool to `connect_to_mem_db`
also please notice how `connect_to_mem_db` is using a ConnectionRegistry. 
This is `Core-Lib` tests. We are testing the `Core-Lib` tools.